config/initializers/omniauth.rb

```diff
@@ -1,4 +1,8 @@
-OmniAuth.config.logger = Rails.logger
+OmniAuth.configure do |config|
+  config.logger = Rails.logger
+  config.path_prefix = "/admin/auth"
+end
+
```

Comment:
I don't think Standard likes this extra line

Reply:
Oops! Fixed, thanks